modules/rostests/apitests/ntdll/NtAllocateVirtualMemory.c

```diff
@@ -600,7 +633,21 @@ CheckSomeDefaultAddresses(VOID)
                                      &Size,
                                      MEM_RESERVE | MEM_COMMIT,
                                      PAGE_READWRITE);
+    // Result varies between Windows versions.
     ok_ntstatus(Status, STATUS_SUCCESS);
+    if (NT_SUCCESS(Status))
```

Comment:
This special-casing unnecessarily bloats the test. I would suggest one of two versions:
* either don't care about other Windows versions, and just make the test succeed on 2003,
* or use something like broken() or ntv6() (see other tests) to distinguish expected results in the same ok() check